docs/tutorials/podman_tutorial.md

```diff
@@ -46,8 +46,8 @@ podman inspect -l | grep IPAddress\":
             "IPAddress": "",
 ```
 
-Note: The -l is a convenience argument for **latest container**.  You can also use the container's ID instead
-of -l.
+Note: The -l is a convenience argument for **latest container**. When using the remote Podman client you have to use the container's ID instead
```

Comment:
Maybe "This option is not available with the remote Podman client, so you have to..."

Reply:
Hi, I've applied your suggestion but using github.dev for the first time I couldn't find out how to squash the commits. Can you take a look please?